public final class com.distil.protection.json.InvalidNumberException extends java.lang.Exception
{
    public static int a;
    public static int b;

    static void <clinit>()
    {
        java.lang.Throwable $r0;
        byte $b3;
        int $i0, $i1, $i2;
        com.distil.protection.json.InvalidNumberException $r1;
        java.lang.NullPointerException r2;

        <com.distil.protection.json.InvalidNumberException: int b> = 1;

        <com.distil.protection.json.InvalidNumberException: int a> = 0;

        $r1 = new com.distil.protection.json.InvalidNumberException;

        specialinvoke $r1.<com.distil.protection.json.InvalidNumberException: void <init>()>();

        $i0 = <com.distil.protection.json.InvalidNumberException: int b>;

        $i1 = -76;

        $i1 = $i0 & $i1;

        $i2 = -1;

        $i2 = $i0 ^ $i2;

        $i2 = $i2 & 75;

        $i1 = $i1 | $i2;

        $i0 = $i0 & 75;

        $b3 = 1;

        $i0 = $i0 << 1;

        $i0 = $i1 + $i0;

        $i1 = $i0 % 128;

        <com.distil.protection.json.InvalidNumberException: int a> = $i1;

        $i0 = $i0 % 2;

        if $i0 == 0 goto label1;

        $b3 = 61;

     label1:
        if $b3 == 61 goto label2;

        return;

     label2:
        r2 = new java.lang.NullPointerException;

        specialinvoke r2.<java.lang.NullPointerException: void <init>(java.lang.String)>("This statement would have triggered an Exception: specialinvoke $u0#15.<java.lang.Object: int hashCode()>()");

        throw r2;

     label3:
        $r0 := @caughtexception;

        throw $r0;

        return;

        catch java.lang.Throwable from label2 to label3 with label3;
    }

    public void <init>()
    {
        com.distil.protection.json.InvalidNumberException r0;

        r0 := @this: com.distil.protection.json.InvalidNumberException;

        specialinvoke r0.<java.lang.Exception: void <init>()>();

        return;
    }
}
